crates/primitives/src/basefee.rs

```diff
@@ -108,4 +108,37 @@ mod tests {
             );
         }
     }
+
+    #[cfg(feature = "optimism")]
+    #[test]
+    fn calculate_base_sepolia_base_fee_success() {
+        let base_fee = [
+            1000000000, 1000000000, 1000000000, 1072671875, 1059263476, 1049238967, 1049238967, 0,
+            1, 2,
+        ];
+        let gas_used = [
+            10000000, 10000000, 10000000, 9000000, 10001000, 0, 10000000, 10000000, 10000000,
+            10000000,
+        ];
+        let gas_limit = [
+            10000000, 12000000, 14000000, 10000000, 14000000, 2000000, 18000000, 18000000,
+            18000000, 18000000,
+        ];
+        let next_base_fee = [
+            1180000000, 1146666666, 1122857142, 1244299375, 1189416692, 1028254188, 1144836295, 1,
+            2, 3,
+        ];
+
+        for i in 0..base_fee.len() {
+            assert_eq!(
+                next_base_fee[i],
+                calc_next_block_base_fee(
+                    gas_used[i] as u128,
+                    gas_limit[i] as u128,
+                    base_fee[i] as u128,
+                    BASE_SEPOLIA_BASE_FEE_PARAMS,
+                ) as u64
+            );
```

Comment:
are these from an op-geth test?

Reply:
Just adjusted from before! This was a copy of the original `calculate_optimism_sepolia_base_fee_success` test above (since it already was setup for using an elasticity multiplier of 10)

And then I changed `calculate_optimism_sepolia_base_fee_success` to expect the correct values when using an elasticity multiplier of 6

Reply:
cool, makes sense